### PR TITLE
Generate Nuklear bindings at build time

### PR DESCRIPTION
**Pull request by 2asoft**
Since bindings are no longer in source control, they are easier to keep up to date (always regenerated on build).
I also took the chance and pointed the Nuklear submodule to the [current repo](https://github.com/Immediate-Mode-UI/Nuklear).

### CODE REVIEW

**Comment by 2asoft** [comment]
I have a followup PR to update to latest version of Nuklear (v4.1.3) that I want to send after this is merged.

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [submodule "nuklear-sys/nuklear-c/nuklear"]
 	path = nuklear-sys/nuklear-c/nuklear
-	url = https://github.com/vurtun/nuklear
+	url = ../../Immediate-Mode-UI/Nuklear
```
Comment (snuk182) [inline .gitmodules:3]: Hm. The repo was moved to https://github.com/Immediate-Mode-UI/Nuklear - why can't we use this updated address to fetch the code?

Comment (2asoft) [reply .gitmodules:3]: Assuming you're asking why the path is relative.

An absolute path such as 'https://github.com/Immediate-Mode-UI/Nuklear' forces git to fetch the submodule via the https protocol. For workflows where ssh protocol is preferred, this means having to manually modify the submodule in the repo.
When a relative path is used, git will use the same protocol for the submodule as the one that was used for cloning the repo, https or ssh.

This StackOverflow answer has a somewhat longer explanation: https://stackoverflow.com/a/44630028/113452.

Comment (2asoft) [reply .gitmodules:3]: @snuk182 do you have any additional thoughts and/or verification that you think would be good to do before this can be merged?

**Pull request merged**